tests/CMakeLists.txt

```diff
@@ -3,7 +3,8 @@ cmake_minimum_required(VERSION 3.10)
 # Link runCSMCameraModelTests with what we want to test and the GTest and pthread library
 add_executable(runCSMCameraModelTests
                PluginTests.cpp
-               FrameCameraTests.cpp)
+               FrameCameraTests.cpp
+               LineScanCameraTests.cpp)
```

Comment:
We should switch to what ISIS does and just glob all of the cpp files in the tests directory.

Reply:
https://cmake.org/cmake/help/v3.13/command/file.html?#filesystem
> 
> Note
> 
> We do not recommend using GLOB to collect a list of source files from your source tree. If no CMakeLists.txt file changes when a source is added or removed then the generated build system cannot know when to ask CMake to regenerate. The CONFIGURE_DEPENDS flag may not work reliably on all generators, or if a new generator is added in the future that cannot support it, projects using it will be stuck. Even if CONFIGURE_DEPENDS works reliably, there is still a cost to perform the check on every rebuild.

Reply:
Is there a better way to manage the list of test sources? This won't be an issue here, but it will be for ISIS.